### PR TITLE
Added sun sensor command and telemetry messages

### PR DESCRIPTION
**Pull request by sakshamkhurana**
## 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have read and followed the `CONTRIBUTING.md` file
- [ ] I have applied the .clang-format file to my changes
- [x] I have commented my code using Doxygen style comments (see Contributing MD for examples), particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the design documentation 
- [ ] I have updated any relevant parameters in the internal ICD
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If this is a bug-fix, I have added tests that can catch similar bugs in the future
- [ ] New and existing unit tests pass locally with my changes
- [ ] For cfe apps I have reserved an empty address for my msgs_ids and perf_ids from the [tracking list](https://docs.google.com/spreadsheets/d/1yRUhNTRBOiGLswlsWftNaYlwGKDeNQ_fCnDYFrc_7fU/edit#gid=772812381) and checked that it does not clash with other apps
- [ ] For new added message types, there is a timestamp included via either an explicit timestamp variable in the struct or buried in the a tlmHeader.

### CODE REVIEW

**Comment by sakshamkhurana** [comment]
Closes #264

File: fsw/public_inc/sunsensor_msg.h [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+/**
+ * @union  sun_sensor_data_t
+ * @brief  Type definition for Sun sensor core data field
+ * @TODO:  get this reviewed if there is a better way to do this
+ * */
+typedef union {
+    unsigned char voltage_data[PHOTO_VOLT_DATA_LEN];  // magic number 16 comes from data sheet where
+                                           // voltages are reported as 4 chunks of 4 bytes each
+    unsigned char app_data[APP_DATA_LEN];  // magic number 9 comes from data sheet where alpha
```
Comment (bkolligs) [inline fsw/public_inc/sunsensor_msg.h:65]: So this is where the alpha and beta angles live?

**Review by bkolligs — COMMENTED** [review]
Perhaps create explicit fields for alpha and beta angles or at least change field names to clearly indicate where these angles are

File: fsw/public_inc/sunsensor_msg.h (hunk 1 above)
Comment (sakshamkhurana) [reply fsw/public_inc/sunsensor_msg.h:65]: I will create a struct for alpha, beta, err code and another one for voltage levels. Then wrap them in the union.

Comment (bkolligs) [reply fsw/public_inc/sunsensor_msg.h:65]: That would be great!

**Comment by sakshamkhurana** [comment]
[Sun sensor message definition](https://trello.com/c/NxtAPHOs/47-sun-sensor-message-definition)

File: fsw/public_inc/sunsensor_msg.h (hunk 1 above)
Comment (sakshamkhurana) [reply fsw/public_inc/sunsensor_msg.h]: @bkolligs Done.

**Review by bkolligs — DISMISSED** [review]

**Review by tenzincrouch — DISMISSED** [review]

**Review by tenzincrouch — APPROVED** [review]

**Pull request merged**